### PR TITLE
feat: added vault data

### PR DESCRIPTION
**Pull request by patcito**
I've also included the script I used. Let me know if all the properties are ok.

### CODE REVIEW

File: data/vaults/0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "crvAAVE",
+  "description": "yearn Curve.fi aDAI/aUSDC/aUSDT",
+  "promoted": true,
+  "retired": false,
+  "migrated": false,
+  "depositDisabled": false,
+  "depositDisabledMessage": "",
+  "withdrawDisabled": false,
+  "withdrawDisabledMessage": false,
+  "apy": { "calculation": "" }
```
Comment (x48-crypto) [inline data/vaults/0x03403154afc09Ce8e44C3B185C82C6aD5f86b9ab.json:12]: @nymmrx do you know what calculation should be for this? Where are you pulling this information today? Also do we need an attribute that describes which harvest number we want to use for APY calculations? Should we add the ability to disable APY calculations?

**Review by x48-crypto — CHANGES_REQUESTED** [review]
@patcito could you rename to "feat: added vault data" please (lowercase the A)?

**Comment by nymmrx** [comment]
@patcito @x48-crypto i think adding **all** the vaults isn't really practical. yearn-meta is meant to be a store of overrides and special cases. keeping a registry in two places in not ideal for a number of reasons. i think we should only keep the ones that need have something special, like a different apy calc or disabled withdrawals / deposits.

what do you think?

**Comment by patcito** [comment]
> @patcito @x48-crypto i think adding **all** the vaults isn't really practical. yearn-meta is meant to be a store of overrides and special cases. keeping a registry in two places in not ideal for a number of reasons. i think we should only keep the ones that need have something special, like a different apy calc or disabled withdrawals / deposits.
> 
> what do you think?

I thought the goal was to move the whole registry to yearn-meta. I'm fine with keeping the current registry, I should add that I'm not familiar with how the current registry is kept by the way. Is there a repo I could look up somewhere? How would we pick the vaults with custom calculation to move to yearn-meta? sorry for all the questions.

**Comment by nymmrx** [comment]
no worries @patcito , registry is kept on chain and will be aggregated by the https://github.com/yearn/yearn-lens contracts for the SDK. yearn-meta is just a registry of all the stuff that is unpractical to store on chain:

- icons
- special vaults overrides
- strategy descriptions
- ...

registry on mainnet is:
- v2.registry.ychad.eth for v2 vaults
- registry.ychad.eth for v1 vaults

**Comment by patcito** [comment]
> no worries @patcito , registry is kept on chain and will be aggregated by the https://github.com/yearn/yearn-lens contracts for the SDK. yearn-meta is just a registry of all the stuff that is unpractical to store on chain:
> 
> * icons
> * special vaults overrides
> * strategy descriptions
> * ...
> 
> registry on mainnet is:
> 
> * v2.registry.ychad.eth for v2 vaults
> * registry.ychad.eth for v1 vaults

ok, let me know which vaults we should keep in there. I'm guessing the migrated and retired ones, and then those with special calculation but not sure which one are these, those with "" as calculation?

**Comment by nymmrx** [comment]
> I'm guessing the migrated and retired ones
yes
> then those with special calculation but not sure which one are these, those with "" as calculation
yes, maybe excluding the ones not endorsed, since we don't show them anyway in v3 ui

**Comment by patcito** [comment]
> yes, maybe excluding the ones not endorsed, since we don't show them anyway in v3 ui

Ok I've only included endorsed ones that are either migrated, retired or with no calculation.

File: schema/vault.json [hunk 2]
```diff
@@ -14,6 +14,24 @@
...
+    },
+    "depositDisabledMessage": {
+      "type": "boolean"
+    },
+    "withdrawDisabled": {
+      "type": "boolean"
+    },
+    "withdrawDisabledMessage": {
+      "type": "boolean"
+    },
```
Comment (nymmrx) [inline schema/vault.json:34]: is this from another PR?

File: data/strategies/0x86eD4F77d40182b8686a25e125FB3f5a04203CaA.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyLenderYieldOptimiser",
+  "description": "This strategy lends the {{tokens}} on various lending platforms such as CREAM, AAVE or ALPHA HOMORA to gain yield.",
+  "author": "Sam Priestley",
+  "authorUri": "https://twitter.com/arbingsam"
+}
```
Comment (nymmrx) [inline data/strategies/0x86eD4F77d40182b8686a25e125FB3f5a04203CaA.json:7]: are all this strategies from another PR?

**Review by nymmrx — CHANGES_REQUESTED** [review]

File: data/strategies/0x86eD4F77d40182b8686a25e125FB3f5a04203CaA.json (hunk 3 above)
Comment (patcito) [reply data/strategies/0x86eD4F77d40182b8686a25e125FB3f5a04203CaA.json]: yes, sorry, just removed this commit from the PR

File: schema/vault.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$schema": "vault",
```
[the remark this answers is not in the document]
Comment (patcito) [reply schema/vault.json]: fixed

**Review by nymmrx — APPROVED** [review]
lgtm

**Pull request merged**